app/javascript/spotlight/admin/crop.js

```diff
@@ -2,22 +2,36 @@ import { addImageSelector } from 'spotlight/admin/add_image_selector'
 import Core from 'spotlight/core'
 
 export default class Crop {
-  constructor(cropArea) {
+  constructor(cropArea, preserveAspectRatio = true) {
     this.cropArea = cropArea;
     this.cropArea.data('iiifCropper', this);
+    // This element will also have the IIIF input elements contained
+    // There may be multiple elements with data-cropper attributes, but
+    // there should only one element with this data-cropper attribute value.
     this.cropSelector = '[data-cropper="' + cropArea.data('cropperKey') + '"]';
     this.cropTool = $(this.cropSelector);
-    this.formPrefix = this.cropTool.data('form-prefix');
-    this.iiifUrlField = $('#' + this.formPrefix + '_iiif_tilesource');
-    this.iiifRegionField = $('#' + this.formPrefix + '_iiif_region');
-    this.iiifManifestField = $('#' + this.formPrefix + '_iiif_manifest_url');
-    this.iiifCanvasField = $('#' + this.formPrefix + '_iiif_canvas_id');
-    this.iiifImageField = $('#' + this.formPrefix + '_iiif_image_id');
-
+    // Exhibit and masthead cropping preserves aspect ratio, while item 
+    // and other widget related cropping does not preserve aspect ratio in the cropping area. 
```

Comment:
I understand what this means but I think it might be made more clear by saying that masthead and exhibit cropping requires a fixed width/height for cropping while items don't.